src/main.rs

```diff
@@ -81,6 +81,7 @@ fn main() {
         force_nullable: args.force_nullable,
         tuple_struct: args.tuple_struct,
         allow_maps_without_value: args.allow_maps_without_value,
+        json_object_path: None,
```

Comment:
This should be passed as a parameter.
How about calling it `json_object_path_regex`?

Reply:
As a parameter to `jsonschema-transpiler`? 
Then we will have to configure it differently in MSG.
With this change as is we parse this config from the schema itself, so every invocation of jsonschema-transpiler, no matter where, will just pick it up (and apparently there's at least a couple of other invocations of jst, e.g. in MPS for testing)

Reply:
> we parse this config from the schema itself, so every invocation of jsonschema-transpiler, no matter where, will just pick it up

I see, I haven't realized that, this makes sense then!